requirements.txt

```diff
@@ -9,3 +9,4 @@ pyyaml==3.11
 python-social-auth==0.2.13
 django-rest-swagger==0.3.4
 django-cors-headers==1.1.0
+git+git://github.com/dlitz/pycrypto@af058ee6f5da391a05275470ab4a4a96aa22b350
```

Comment:
ok.....sounds like it is actually dead and there is a forked project that is well maintained that others are using. I'll look into it on Monday and see if that would be a good solution for us. Not sure why I didn't see that sooner.
https://pypi.python.org/pypi/pycryptodome